lib/modules/manager/helmfile/extract.ts

```diff
@@ -18,8 +18,13 @@ import {
   localChartHasKustomizationsYaml,
 } from './utils';
 
-const isValidChartName = (name: string | undefined): boolean =>
-  !!name && !regEx(/[!@#$%^&*(),.?":{}/|<>A-Z]/).test(name);
+const isValidChartName = (name: string | undefined, oci: boolean): boolean => {
```

Comment:
```suggestion
function isValidChartName(name: string | undefined, oci: boolean): boolean {
```
while changing move to ordinary function declaration

Reply:
Done in 4f73509